subworkflows/local/prepare_genome.nf

```diff
@@ -18,7 +18,7 @@ workflow PREPARE_GENOME {
 
         ch_bwamem2_version = Channel.empty()
         // Fetch BWAMEM2 index or create from scratch if required
-        if ( params.bwamem2 && file(params.bwamem2, checkIfExists:true) ) {
+        if ( params.bwamem2_index && file(params.bwamem2_index, checkIfExists:true) ) {
```

Comment:
We should have a check to see if the aligner is set to bwa. Otherwise we don't need to build the index